client/express-checkout/utils/index.ts

```diff
@@ -108,3 +111,70 @@ export const getErrorMessageFromNotice = ( notice: string ) => {
 	div.innerHTML = notice.trim();
 	return div.firstChild ? div.firstChild.textContent : '';
 };
+
+/**
+ * Returns the appearance settings for the Express Checkout buttons.
+ * Currently only configures border radius for the buttons.
+ */
+export const getExpressCheckoutButtonAppearance = () => {
+	const buttonSettings = getExpressCheckoutData( 'button' );
+
+	return {
+		variables: { borderRadius: `${ buttonSettings?.radius ?? 10 }px` },
```

Comment:
When ECE is disabled that implies that the Payment Request Buttons are enabled instead. Those do not support changing the border radius, so we need a fallback value when ECE is disabled.

A default radius of 10px was something I just chose arbitrarily while working on the PR. We'll probably want some other default radius for when ECE is disabled, likely to match what the PRBs use.

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/9010/commits/b76d492b54b2137d1193677bfc225d9e5dd0658e